pkg/statistics/handle/autoanalyze/priorityqueue/dynamic_partitioned_table_analysis_job.go

```diff
@@ -204,6 +204,7 @@ func (j *DynamicPartitionedTableAnalysisJob) analyzePartitionIndexes(
 ) {
 	analyzePartitionBatchSize := int(variable.AutoAnalyzePartitionBatchSize.Load())
 
+OnlyPickOneIndex:
```

Comment:
The reasons I use a label here are:
1. I can't easily retrieve the first kv from a map, so I prefer to use a for loop.
2. If I directly return from the inner loop, I will get an SA4004 warning from the static check.

So I think a named label would help.